test/src/tests-persistence.ts

```diff
@@ -652,6 +659,8 @@ describe('migrations and persistence-related', () => {
             JSON.stringify({ mpid: 'mpid1', is_logged_in: false }),
         ]);
 
+        // tick clock forward to go beyond the 1 day identity caching
+        clock.tick(MILLISECONDS_IN_ONE_DAY_PLUS_ONE_SECOND);
```

Comment:
the clock.ticks can probably be removed now that we are hiding this behind a feature flag